packages/office-ui-fabric-react/src/components/Button/Button.Props.ts

```diff
@@ -76,11 +76,16 @@ export interface IButtonProps extends React.HTMLAttributes<HTMLButtonElement | H
 
   /**
    * Props for button menu. Providing this will default to showing the menu icon. See menuIconProps for overriding
-   * how the default icon looks. Providing this in addition of onClick will render a SplitButton.
+   * how the default icon looks. Providing this in addition of onClick and setting the split property to true will render a SplitButton.
    */
   menuProps?: IContextualMenuProps;
 
   /**
+   * If set to true, and if menuProps and onClick are provided, the button will render as a SplitButton. Defaults to false.
+   */
+  split?: boolean;
```

Comment:
shall we warn if the user provides this but not the others? Ideally we would not have them as optional...like using union types.